src/test/java/nextstep/ladder/util/StringParserTest.java

```diff
@@ -0,0 +1,23 @@
+package nextstep.ladder.util;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class StringParserTest {
+
+    @Test
+    @DisplayName("성공 - 참여자의 이름을 쉼표로 구분한다.")
+    void success_string_parse() {
+        String text = "pobi,honux,crong,jk";
+        int size = 4;
+
+        List<String> persons = StringParser.split(text);
+
+        assertThat(persons).hasSize(size);
```

Comment:
```suggestion
        assertThat(persons).isEqualTo(List.of("pobi", "honux", "crong", "jk"));
```

저는 이렇게 할것 같아요 🤔 
객체간의 비교를하면 전반적으로 모든 조건을 테스트 할 수 있을것 같아서요 😄